src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1

```diff
@@ -0,0 +1,154 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+<#
+.Synopsis
+Migrate logins from the source Sql Servers to the target Azure Sql Servers.
+.Description
+Migrate logins from the source Sql Servers to the target Azure Sql Servers.
+#>
+
+
+function New-AzDataMigrationLoginsMigration
+{
+    [OutputType([System.Boolean])]
+    [CmdletBinding(PositionalBinding=$false)]
+    [Microsoft.Azure.PowerShell.Cmdlets.DataMigration.Description('Migrate logins from the source Sql Servers to the target Azure Sql Servers.')]
```

Comment:
`New-AzDataMigrationLoginsMigration` Does not support ShouldProcess but the cmdlet verb New indicates that it should.